include/eve/algo/copy.hpp

```diff
@@ -16,11 +16,11 @@
 namespace eve::algo
 {
   //================================================================================================
-  //! @addtogroup eve.algo
+  //! @addtogroup algo
   //! @{
   //!  @var copy
   //!
-  //!  @brief version of std::copy
+  //!  @brief SIMD version of std::copy
```

Comment:
You want to call these SIMD versions? Hmm. All of our library is SIMD. A bit weird.